recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 build:
   number: 1
   skip: true  # [py<38 or win32 or s390x or py>38 and not test_py39]
```

Comment:
Note that while s390x builds may be stuck, this package skips that platform by design.

Reply:
(we still need osx-64 to finish building however, at minimum.)